<?xml version="1.0"?>
<launch>
		<!--env name="ROSCONSOLE_CONFIG_FILE" value="$(find sofar_hbc_12)/config/console.config" /-->
		
    <node name="blocks_server" pkg="sofar_hbc_12" type="blocks_server" args="--wait"/> 
    <rosparam file="$(find sofar_hbc_12)/config/blocks_param.yaml" command="load"/>
    
    <param name="baxter_description"
      command="$(find xacro)/xacro $(find baxter_description)/urdf/baxter.urdf.xacro" />
    
    <param name="human_description" command="$(find xacro)/xacro $(find sofar_hbc_01)/urdf/human.urdf.xacro"/>
    <node name="closest_block" pkg="sofar_hbc_12" type="closest_block" output="screen" args="--wait" />
    
    <node name="empty_pos_server" pkg="sofar_hbc_12" type="empty_pos_server" output="screen" args="--wait" />
    
    <node name="finite_state_machine_left" pkg="sofar_hbc_12" type="finite_state_machine" args="--wait" output="screen" respawn="false" required="true">
    	<param name="~arm" value="left"/>	
    </node>
    <node name="finite_state_machine_right" pkg="sofar_hbc_01" type="finite_state_machine" args="--wait" output="screen" respawn="false" required="true">
    	<param name="~arm" value="right"/>
    </node>
    
</launch>
